_posts/languages/php/2000-01-01-laravel.md

```diff
@@ -39,6 +39,14 @@ defined with a random string. The `artisan` script can generate one for you:
 scalingo env-set APP_KEY=$(php artisan --no-ansi key:generate --show)
 ```
 
+Then, you will need to set the `APP_URL` environment variable to your main domain
+(for instance, https://yourapp.osc-fr1.scalingo.io), otherwise you may end up
```

Comment:
We used `my-app` as example name in all the documentation.

```suggestion
(for instance, https://my-app.osc-fr1.scalingo.io), otherwise you may end up
```